swtbot_test/src/iotLTStestsuites/CodeGeneration.java

```diff
@@ -111,7 +111,7 @@ public void tc_02_GenerateDefault() throws Exception {
 
 		Utility.clickGenerateCode();
 
-		String task = "ret = xTaskCreate(task_1, \"task_1\", 512, NULL, 1, NULL)";
+		String task = "ret = xTaskCreate(task_2, \"task_2\", 512, NULL, 1, NULL)";
```

Comment:
Q: Why we need to change this parameter?

Reply:
So the scripts are not flexible for multiple packages. we need to consider this since we cannot modify the script each time a new package is released or updated

Reply:
I have created new ticket to handle this flexibility: RSC-17994